nodes/mainnet.md

```diff
@@ -99,20 +99,25 @@ RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with
 - `public-celestia-rpc.numia.xyz`
 - `celestia-rpc.mesa.newmetric.xyz`
 - `rpc.lunaroasis.net`
+- `https://celestia-rpc.brightlystake.com`
 
 ### API endpoints
 
 - `public-celestia-lcd.numia.xyz`
 - `celestia-rest.mesa.newmetric.xyz`
 - `api.celestia.pops.one`
 - `api.lunaroasis.net`
+- `https://celestia-rpc.brightlystake.com/api`
 
 ### gRPC endpoints
 
 - `public-celestia-grpc.numia.xyz`
 - `celestia-grpc.mesa.newmetric.xyz`
 - `grpc.celestia.pops.one`
 - `grpc.lunaroasis.net:443`
+- `celestia-rpc.brightlystake.com:9090`
+
+You can find the status of these endpoints [here](https://celestia-tools.brightlystake.com/)
```

Comment:
```suggestion
```

Reply:
removing, as it is not for the status of all of "these" endpoints on the page, just brightlystake

Reply:
<img width="1435" alt="Screenshot 2023-11-01 at 20 00 39" src="https://github.com/celestiaorg/docs/assets/123407053/f555288a-3354-4647-8945-e693d74bf6de">
We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.

Reply:
@jcstein

Reply:
right now we have url fetched every hour automatically, rpc and grpc endpoints are active

Reply:
"bud" okay...thanks for the context.

Reply:
> We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.
got it. thanks bud...

it does not seem to show all endpoints, though, just the consensus endpoints. is this meant to be a dashboard to show [da rpc endpoints](https://docs.celestia.org/nodes/mainnet#da-rpc-endpoints) or [consensus rpc endpoints](https://docs.celestia.org/nodes/mainnet#consensus-rpc-endpoints)? it appears its showing just consensus for mainnet

Reply:
> We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.
got it. thanks...

it does not seem to show all endpoints, though, just the consensus endpoints. is this meant to be a dashboard to show [da rpc endpoints](https://docs.celestia.org/nodes/mainnet#da-rpc-endpoints) or [consensus rpc endpoints](https://docs.celestia.org/nodes/mainnet#consensus-rpc-endpoints)? it appears its showing just consensus for mainnet, but is listed under gRPC endpoints

Reply:
could you maybe show different columns with the "type" of endpoint perhaps?

Reply:
I have now made changes to 
- test for http and https
- with respect to the da rpc end point vs consensus 
we can identify with INDEXER = ON and EARLIEST BLOCK = 1 

i have added that as a note bud. Hope this is fine.

Reply:
@jcstein please take a look once and correct me if my understanding is incorrect

Reply:
LGTM!